mlir/test/rocmlir-driver/populate_subkernels.mlir

```diff
@@ -1,21 +1,21 @@
-// RUN: rocmlir-gen -conv-config "--operation conv2d --arch amdgcn-amd-amdhsa:gfx906 --num_cu 64 --fil_layout GNCHW --in_type fp32 --fil_type fp32 --out_type fp32 --in_layout NGCHW --out_layout NGCHW --batchsize 64 --in_channels 1024 --out_channels 1024 --in_h 14 --in_w 14 --out_h 14 --out_w 14 --fil_h 1 --fil_w 1 --dilation_h 1 --dilation_w 1 --conv_stride_h 1 --conv_stride_w 1 --padding_h 0 --padding_w 0 --kernel_name conv2d_fwd --groupsize 1 --kernel_id 0" | FileCheck %s --check-prefix=KERNEL0
-// RUN: rocmlir-gen -conv-config "--operation conv2d --arch amdgcn-amd-amdhsa:gfx906 --num_cu 64 --fil_layout GNCHW --in_type fp32 --fil_type fp32 --out_type fp32 --in_layout NGCHW --out_layout NGCHW --batchsize 64 --in_channels 1024 --out_channels 1024 --in_h 14 --in_w 14 --out_h 14 --out_w 14 --fil_h 1 --fil_w 1 --dilation_h 1 --dilation_w 1 --conv_stride_h 1 --conv_stride_w 1 --padding_h 0 --padding_w 0 --kernel_name conv2d_fwd --groupsize 1 --kernel_id 1" | FileCheck %s --check-prefix=KERNEL1
-// RUN: rocmlir-gen -conv-config "--operation conv2d --arch amdgcn-amd-amdhsa:gfx906 --num_cu 64 --fil_layout GNCHW --in_type fp32 --fil_type fp32 --out_type fp32 --in_layout NGCHW --out_layout NGCHW --batchsize 64 --in_channels 1024 --out_channels 1024 --in_h 14 --in_w 14 --out_h 14 --out_w 14 --fil_h 1 --fil_w 1 --dilation_h 1 --dilation_w 1 --conv_stride_h 1 --conv_stride_w 1 --padding_h 0 --padding_w 0 --kernel_name conv2d_fwd --groupsize 1 --kernel_id 2" | FileCheck %s --check-prefix=KERNEL2
-// RUN: rocmlir-gen -conv-config "--operation conv2d --arch amdgcn-amd-amdhsa:gfx906 --num_cu 64 --fil_layout GNCHW --in_type fp32 --fil_type fp32 --out_type fp32 --in_layout NGCHW --out_layout NGCHW --batchsize 64 --in_channels 1024 --out_channels 1024 --in_h 14 --in_w 14 --out_h 14 --out_w 14 --fil_h 1 --fil_w 1 --dilation_h 1 --dilation_w 1 --conv_stride_h 1 --conv_stride_w 1 --padding_h 0 --padding_w 0 --kernel_name conv2d_fwd --groupsize 1 --kernel_id 3" | FileCheck %s --check-prefix=KERNEL3
```

Comment:
Sorry for jumping in. If the `num_cu` is optional, do you have to remove them? If the client (MIOpen) usually provide this attribute, maybe you also want to keep it here.

Reply:
You do want to keep them inside the --conv-config arguments, since MIOpen will always pass them.

For `--conv-config` tests, you should probably put those back

Good catch, LIxun

Reply:
I see, it makes perfect sense, thanks!